tests/Tester.py

```diff
@@ -0,0 +1,42 @@
+import csv
+
+from apis import WikipediaApi
+from src import HeuristicTester
+from src.heuristics import Heuristics
+
+test_file_path = "tests.txt"
+results_file_path = "results.tsv"
+
+api = WikipediaApi.WikipediaApi()
+heuristics = [Heuristics.BfsHeuristic(), Heuristics.DfsHeuristic()]
+
+
+def run_tests():
+    tests = []
+
+    with open(test_file_path, "r") as test_file:
+        for line in test_file:
+            if not line.startswith("#"):
+                parts = line.rstrip().split("|")
+                tests.append((parts[0], parts[1]))
+
+    with open('results.tsv', 'w', newline='') as f:
+        writer = csv.writer(f, delimiter='\t',)
+        writer.writerow(["start", "goal", "heuristic", "time_seconds", "nodes_expanded", "path_length", "path"])
+        for test in tests:
+            start, goal = test
+            writer.writerows(run_one_test(start, goal))
+
+
+def run_one_test(start, goal):
+    res = HeuristicTester.HeuristicTester.compare_heuristics(start, goal, api, heuristics)
+    ans_list = []
+    for r in res:
+        ans_list.append(
+            [start, goal, r["heuristic"], str(r["time_seconds"]), str(r["nodes_expanded"]), str(r["path_length"]),
```

Comment:
You can `map(str, [list...])` here to clean things up if you want #functionalprogramming